frame/instrumental-strategy-pablo/src/mock/helpers.rs

```diff
@@ -0,0 +1,35 @@
+use composable_traits::defi::CurrencyPair;
+use frame_support::assert_ok;
+use pallet_pablo::PoolInitConfiguration;
+use primitives::currency::CurrencyId;
+use sp_runtime::Permill;
+
+use super::runtime::{Balance, Pablo, PoolId};
+use crate::mock::runtime::{AccountId, BlockNumber};
+
+pub fn create_usdt_usdc_pool() -> PoolId {
+	let usdt_amount = 1_000_000_000 * CurrencyId::unit::<Balance>();
+	let usdc_amount = 1_000_000_000 * CurrencyId::unit::<Balance>();
+	let assets = CurrencyPair::new(CurrencyId::USDT, CurrencyId::USDC);
+	let amounts = vec![usdt_amount, usdc_amount];
+	create_pool(assets, amounts, 100, Permill::zero())
+}
+
+fn create_pool(
+	assets: CurrencyPair<CurrencyId>,
+	// TODO(saruman9): add amount to a pools
+	_amounts: Vec<Balance>,
+	amplification_coefficient: u16,
+	fee: Permill,
+) -> PoolId {
+	let config = PoolInitConfiguration::<AccountId, CurrencyId, BlockNumber>::StableSwap {
```

Comment:
Not really. I chose almost random values just to fix a specific test (`rebalance_emits_event` which doesn't need concrete values). For the rest of tests, it may be necessary to create other pools, assets, etc.

Reply:
I got it. It seems to me, as we discussed, for MVP that all manipulations will occur only with `Uniswap` pool, so maybe it should be changed.